spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,62 @@
+describe 'User stories' do
+  subject(:takeaway) {Takeaway.new({menu_klass: Menu.new, sendSms_klass: SendSms.new})}
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.